src/index.css

```diff
@@ -1,27 +1,33 @@
-body {
+* {
```

Comment:
- [ ] As requested by the previous reviewer, please make your calculator _match_ the given design.

In this review iteration, you are mostly matching the colors, but not the widths, the alignments, the padding, the margins.


Your Implementation | Given Design
:---:|:---:
![image](https://user-images.githubusercontent.com/57421823/158539956-39c341c9-3b1b-42b7-b497-29b7eae5411c.png) | ![image](https://user-images.githubusercontent.com/57421823/158540036-d6f74970-10ce-4f7d-be46-6d3f3afa2220.png)

I recommend using flexbox to align your calculator, so it's easier.

Please, kindly make your calculator match the given design.